### PR TITLE
Use proper files for cron jobs

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: cron/cron.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  console.log('console.debug is not active - thanks to ACTIVATE_DEBUG_LOG env variable');
+  console.debug = () => {};
+}
+
+sentry.initCaptureConsole();
+
+const jobs = activation
+  .concat(automation)
+  .concat(demarchesSimplifiees)
+  .concat(summary);
```
Comment (sandrica89) [inline cron/cron.ts:20]: I like having all definitions in one place, to check the timing of jobs (to plan releases for example).
I would use one definition file for all jobs

File: cron/jobs/psychologists.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
-import psychologists from '../db/psychologists';
+import psychologists from '../../db/psychologists';
 
-const reactivatePsychologists = async () : Promise<void> => {
+const reactivatePsychologists = async () : Promise<boolean> => {
   const results = await psychologists.reactivate();
```
Comment (sandrica89) [inline cron/jobs/psychologists.ts:4]: We could catch any error here and return false in that case.

**Review by sandrica89 — CHANGES_REQUESTED** [review]
* `Procfile` and `README.md` needs to be updated
* Not sure to understand what `cron/test.ts` is testing :thinking:

**Comment by desoindx** [comment]
After discussion, useless refacto

**Pull request closed without merging**